content/en/blog/releases/release_v0.6.0.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Announcing OpenFunction 0.6.0: FaaS observability, HTTP trigger, and more"
+linkTitle: "Release v0.6.0"
+date: 2022-03-25
+weight: 95
+
+---
+
+[OpenFunction](https://github.com/OpenFunction/OpenFunction) is a cloud-native open source FaaS platform aiming to let you focus on your business logic. The OpenFunction community has been hard at work over the past several months preparing for the v0.6.0 release. Today, we proudly announce v0.6.0 is officially available. Thanks to the community for helping drive the new features, enhancements, and bug fixes.
```

Comment:
Both v0.6.0 and 0.6.0 are ok, but it'll be good to use one of them in one article instead of using both

Reply:
@benjaminhuo Updated.